src/AddListItem.stories.js

```diff
@@ -0,0 +1,11 @@
+import AddListItem from './AddListItem';
+
+export default {
+  title: 'Component/AddListItem',
+  component: AddListItem,
+};
+
+const Template = args => <AddListItem {...args} />;
```

Comment:
If you don't use 'args' or don't have any, it's not necessary to write it this way.
Just in case there are more versions of 'AddListItem' 😅

Reply:
That's true, thanks.